index.html

```diff
@@ -228,5 +231,51 @@ <h4>Skills</h4>
       
     </section>
 
+
+    <!-- Form section -->
+
+    <footer>
+        <p class="get-in-touch">
+            I'm always interested in hearing about new projects, 
+            so if you'd like to chat please get in touch.
+        </p>
+
+        <form action="https://formspree.io/f/mpzbgqgr" method="POST">
+            <ul>
+                <li>
+                    <input type="text" id="name" name="text" maxlength="30" placeholder="Full Name" class="full-name" required>
+                    <label for="name"></label>
+                </li>
+                <li>
+                    <input type="email" id="email" name="email" placeholder="Email Address" class="email" required>
+                    <label for="email"></label>
+                </li>
+                <li>
+                    <textarea name="comment" maxlength="500" placeholder="Enter text here" cols="60" rows="30" class="textarea" required>
+
+                    </textarea>
+                </li>
+                <li>
+                    <div class="btn form-btn">
+                        <button class="button" type="submit">Get in touch</button>
+                     </div>
+                </li>
+            </ul>
```

Comment:
- [ ] Your design looks good, I must commend you 🤝 . I think your UI can be better if you use flexbox to layout the elements in the form. This is to make the elements responsive both on mobile and desktop. Please add it 👍 .